ligo/src/startup.mligo

```diff
@@ -100,6 +100,19 @@ let storable_get_vote_permit_counter (packed_param, config_store : bytes * confi
     end
   | None -> (failwith "ENTRYPOINT_NOT_FOUND" : return)
 
+let storable_get_total_supply (packed_param, config_store : bytes * configured_storage) : return =
```

Comment:
This should be mentioned as one of the entrypoints that doesn't directly traslate in the "module" comment on top.

Reply:
I've updated this, as well as other changes, this should be good now.